src/test/java/org/terasology/flexiblepathfinding/helpers/TestPaths.java

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.flexiblepathfinding.helpers;
+
+public final class TestPaths {
+    public static final String[] singleFlatStepNorthPath = {
```

Comment:
Checkstyle wants you to write this as 
```suggestion
    public static final String[] SINGLE_FLAT_STEP_NORTH_PATH= {
```

not sure this is better... maybe we should disable checkstyle for the whole file?

Reply:
I'm okay with both